offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
```

Comment:
The version is a bit confusing. While I see where it comes from, some might expect it to be the same as the projects version `1.2.0`. 
What do you think?